### ISSUE TITLE
Enhancement - List of Wanted cards

### ISSUE THREAD

**Issue by Ansem93** [ko-vasilev/Hearthstone-Collection-Tracker#50]
Hi, i was thinking about to create a list of wanted cards.
Like when i'm looking into my missing cards, i can click once or twice for add the cards that i want into a list.
This way it would be easier to understand which is the collection that we must buy!

**Comment by ko-vasilev** [comment]
There is already a "Desired cards" feature. Doesn't it kind of do the same thing? Although right now you cannot view all your desired cards at once.

**Comment by Ansem93** [comment]
The desired card are simply the missing one, If I want to remove them to the list, it will add them to my collection.
I want instead a shorter list, with only the card that I really want.
I think that it would be easy to do: like a checkbok on the right than the missing counter, and when I check it would add the cart to this different list.

**Comment by ko-vasilev** [comment]
Yep, "Desired cards" thing does exactly this. You can even see amount of missing cards you want. (How many copies of each card can be set from Manage Set Collection -> combo box near each card).

**Comment by Ansem93** [comment]
the problem is: if i try to delete the card from the desired card, it will add them to my collection.
Can you please disable that if i delete a card from the desired list, it won't add it to my collection?

**Comment by ko-vasilev** [comment]
I don't understand what you're mean. You cannot delete card from "desired" cards, you can only delete cards from your collection. With dropdown list you just specify how many copies of a card you would like to have in your collection.
